pkg/formatted/k8s.go

```diff
@@ -30,6 +30,7 @@ var ConditionColor = map[string]color.Attribute{
 	"Cancelled": color.FgHiYellow,
 	"Completed": color.FgHiMagenta,
 	"Pending":   color.FgHiWhite,
+	"Started":   color.FgHiCyan,
```

Comment:
This color is a little tough to see on a white background. Anything else we could use?

![image](https://user-images.githubusercontent.com/34258252/91190748-8866d400-e6b9-11ea-8521-54eb9d5718c3.png)

Reply:
Should I mark the same color as Running? The only remaining one form there was Black and Cyan

Reply:
Yeah, I think for now we can go with the same Running color and update later one if we would like to distinguish.

Reply:
done